src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,17 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        User[] users = UserService.getUsers();
+
+        for (User user : users) {
+            String emailBuffer = user.getEmail();
```

Comment:
you should create a new object of UserService

and call its method userService.findByEmail() to find user by email